--- cinnamon-desktop-5.8.0/schemas/org.cinnamon.desktop.background.gschema.xml.in.orig	2023-06-01 14:56:48.000000000 +0200
+++ cinnamon-desktop-5.8.0/schemas/org.cinnamon.desktop.background.gschema.xml.in	2023-06-13 19:30:23.558504820 +0200
@@ -11,7 +11,7 @@
       </description>
     </key>
     <key name="picture-uri" type="s">
-      <default>'file://@datadir@/themes/Adwaita/backgrounds/adwaita-timed.xml'</default>
+      <default>'file:///usr/share/backgrounds/gnome/adwaita-l.webp'</default>
       <summary>Picture URI</summary>
       <description>
         URI to use for the background image. Not that the backend only supports
