--- cinnamon-desktop-5.8.0/schemas/org.cinnamon.desktop.screensaver.gschema.xml.in.orig	2023-06-13 18:18:10.638644921 +0200
+++ cinnamon-desktop-5.8.0/schemas/org.cinnamon.desktop.screensaver.gschema.xml.in	2023-06-13 18:18:13.098631594 +0200
@@ -77,21 +77,21 @@
       </description>
     </key>
     <key name="font-date" type="s">
-      <default>"Ubuntu 24"</default>
+      <default>"Cantarell 24"</default>
       <summary>The date time font</summary>
       <description>
         This is the font that the date uses for the screen locker.
       </description>
     </key>
     <key name="font-time" type="s">
-      <default>"Ubuntu 64"</default>
+      <default>"Cantarell 64"</default>
       <summary>The time font</summary>
       <description>
         This is the font that the time uses for the screen locker.
       </description>
     </key>
     <key name="font-message" type="s">
-      <default>"Ubuntu 14"</default>
+      <default>"Cantarell 14"</default>
       <summary>The away message font</summary>
       <description>
         This is the font that the away message use for the screen locker.
